diff --git a/tunnel/statistic/manager.go b/tunnel/statistic/manager.go
index f38ed07..39cc364 100644
--- a/tunnel/statistic/manager.go
+++ b/tunnel/statistic/manager.go
@@ -93,6 +93,14 @@ func (m *Manager) Total() (up, down int64) {
 	return m.uploadTotal.Load(), m.downloadTotal.Load()
 }
 
+func (m *Manager) ProxyNow() (up, down int64) {
+	return m.proxyUploadBlip.Load(), m.proxyDownloadBlip.Load()
+}
+
+func (m *Manager) ProxyTotal() (up, down int64) {
+	return m.proxyUploadTotal.Load(), m.proxyDownloadTotal.Load()
+}
+
 func (m *Manager) Memory() uint64 {
 	m.updateMemory()
 	return m.memory
